Lib/test/support/strace_helper.py

```diff
@@ -163,6 +164,13 @@ def requires_strace():
     if sys.platform != "linux":
         return unittest.skip("Linux only, requires strace.")
 
+    if 'LD_PRELOAD' in os.environ:
```

Comment:
```suggestion
    if "LD_PRELOAD" in os.environ:
```

I think it's better to strictly follow PEP 8 in new code.

Reply:
https://peps.python.org/pep-0008/#string-quotes neither is actually PEP-8 preferred, but yes, most the code in this uses double quotes so reasonable to move this to double quotes